app/src/main/webapp/app/error/error.ctrl.js

```diff
@@ -170,5 +170,54 @@ esbMessageAdminApp.controller('ErrorsSearchCtrl', [
       $scope.calendarToOpened = true;
     };
     // End Date picker stuff
+
+    // on message select, fetch message details
+    $rootScope.$watch('selectedMessage',
```

Comment:
Actually, better yet, let's just move the body of this watch callback to the watch for "messageSelections". If messageSelections.length > 0, then do lines 178 to 212. That is, query for the selected message id and populate `$scope.message` with the result. Then we can remove this watch callback on `selectedMessage` entirely.